### ISSUE TITLE
Set Up Travis-CI

### ISSUE THREAD

**Issue by esoterik** [YaleSTC/roomer#4]
Blocked on #2

Should migrate style checking script from [Reservations](https://github.com/YaleSTC/reservations/issues/1581)

**Comment by orenyk** [comment]
We need to disable notifications, I'll update and amend

**Comment by orenyk** [comment]
I'm going to repurpose this just for Travis in general and open a new issue for adding rubocop to our builds. Opening a PR too.
